### PR TITLE
Stop supporting elastic-app-search in favour of enterprise-search and update dependencies 

### PR DESCRIPTION
**Pull request by flara-infoxchange**
Drop elastic-app-search in favor of only using elastic-enterprise-search and update dependencies

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -401,9 +401,9 @@ We use the official `elastic app search python client <https://github.com/elasti
```
Comment (michaeladams) [inline README.rst:401]: The docs have a link above this to app-search-python - which is the deprecated library - could you update it so it points to https://github.com/elastic/enterprise-search-python?

Comment (flara-infoxchange) [reply README.rst:401]: Done

Comment (michaeladams) [reply README.rst:401]: Theres the second link that still points to the old repo.

Comment (flara-infoxchange) [reply README.rst:401]: It should be ok now

Comment (michaeladams) [reply README.rst:401]: thanks!

**Review by michaeladams — APPROVED** [review]

**Review by michaeladams — APPROVED** [review]

**Pull request merged**